batchlib/analysis/cell_level_analysis.py

```diff
@@ -199,109 +182,178 @@ def denoise(self, img):
         return skimage.morphology.opening(img, selem=self.structuring_element)
 
 
-class CellLevelAnalysis(BatchJobWithSubfolder):
-    """
-    """
+class InstanceFeatureExtraction(BatchJobWithSubfolder):
```

Comment:
This is computing the features per cell, correct?
Can we use `BatchJobOnContainer` and then write these to a hdf5 table?

Reply:
Correct, and yes, I am on it.